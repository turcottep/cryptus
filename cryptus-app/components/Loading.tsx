import React from "react";

export default function Loading() {
  return (
    <div
      id="header"
      className="w-full h-full flex justify-around items-center bg-opacity-80 bg-black"
    >
      <div className="flex flex-col">
        <img
          className="rounded-full border-2 bg-white border-black w-24 h-24"
          src="../icons/icon-192x192.png"
          alt="PublicWallet Icon"
        />
        <span className=" text-white text-2xl font-bold">Loading ...</span>
      </div>
    </div>
  );
}
