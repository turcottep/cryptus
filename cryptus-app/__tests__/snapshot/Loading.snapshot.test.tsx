// components/__tests__/button.snapshot.test.tsx
import React from 'react';
import renderer from 'react-test-renderer';
import Loading from "../../components/Loading"

it('renders correctly', () => {
  const tree = renderer.create(<Loading/>).toJSON()
  expect(tree).toMatchSnapshot()
})