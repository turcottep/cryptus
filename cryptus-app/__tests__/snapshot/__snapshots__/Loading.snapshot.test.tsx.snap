// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=" w-screen h-screen flex justify-around items-center bg-opacity-80 bg-black"
  id="header"
>
  <div
    className="flex flex-col"
  >
    <img
      alt="PublicWallet Icon"
      className="rounded-full border-2 bg-white border-black w-24 h-24"
      src="../icons/icon-192x192.png"
    />
    <span
      className=" text-white text-2xl font-bold"
    >
      Loading ...
    </span>
  </div>
</div>
`;
